src/Esfa.Vacancy.Register.Infrastructure/Settings/MachineSettings.cs

```diff
@@ -7,7 +7,9 @@ public sealed class MachineSettings : IProvideSettings
     {
         public string GetSetting(string settingKey)
         {
-            return Environment.GetEnvironmentVariable($"DAS_{settingKey.ToUpper(CultureInfo.InvariantCulture)}", EnvironmentVariableTarget.User);
+            return Environment.GetEnvironmentVariable(
+                $"DAS_{settingKey.ToUpper(CultureInfo.InvariantCulture)}",
+                EnvironmentVariableTarget.Machine);
```

Comment:
Why has this been changed?

Reply:
To allow debug while hosted in IIS

Reply:
I'd definitely prefer not to have to do this, having to host in IIS would be a backward step for the solution.